packages/model-viewer/src/test/decorators-spec.ts

```diff
@@ -50,7 +50,7 @@ suite('decorators', () => {
       customElements.define(tagName, class extends StyleableElement {});
 
       element = document.createElement(tagName) as StyleableElement;
-      document.body.appendChild(element);
+      document.body.insertBefore(element, document.body.firstChild);
```

Comment:
What's all this about? Something browser-specific?

Reply:
Now that we are running in vanilla mocha, without additional intervention all of our elements get appended after the Mocha test results. This ensures that our test elements are inserted before those Mocha DOM elements.